application/models/user_access_token.go

```diff
@@ -79,26 +79,30 @@ func (u *UserAccessToken) FindByBearerToken(bearerToken string) error {
 }
 
 // GetOrganization returns the Organization of the UserOrganization of the UserAccessToken
-//  I'm not using DB.Eager() or DB.Load() because they seemed to keep hanging.
-//  I wonder if it was because of the has_many  <--->  belongs_to relations
 func (u *UserAccessToken) GetOrganization() (Organization, error) {
 	if u.UserOrganizationID <= 0 {
 		return Organization{}, fmt.Errorf("user access token id %v has no user organization", u.ID)
```

Comment:
Is this check necessary? (same a few lines down)

Reply:
OK. I was worried about panics, but it looks like your right about Load catching it.